src/cli/cmds/device.toit

```diff
@@ -31,7 +31,10 @@ EXTRACT-FORMATS-COMMAND-HELP ::= """
     - 'binary': a binary image suitable for OTA updates.
     - 'tar': a tar file with the device firmware. Only available for host devices.
       That is, devices that use an envelope built for Linux, macOS, or Windows.
-    - 'qemu': a QEMU image. Only available for ESP32 devices.
+    - 'full-image': a full binary image suitable for QEMU, Wokwi, or flashing
```

Comment:
We don't have partial images, so maybe this should just be image? Or are you worried that it conflicts with program images and container images?

Reply:
I think I had 'full-binary' first, and thus needed to distinguish from "binary".
"image" sounds good. We should probably avoid using the term "container image" to avoid confusion.
done.